client/gui/translations/ws_desktop_de.ts

```diff
@@ -71,7 +71,7 @@
     </message>
     <message>
         <source>Connect to rate</source>
-        <translation>Verbinden Sie sich mit dem Tarif</translation>
+        <translation>Verbinden Sie sich zum bewerten</translation>
```

Comment:
Tarif is something like a Mobile contract or even the 10/30gb limits of the free plans
Bewerten = Review / Rate